content/riak/kv/2.2.0/using/repair-recovery/rolling-replaces.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Rolling Replaces"
+description: ""
+project: "riak_kv"
+project_version: "2.2.0"
+menu:
+  riak_kv-2.2.0:
+    name: "Rolling Replaces"
+    identifier: "repair_recover_replace"
+    weight: 106
+    parent: "managing_repair_recover"
+toc: true
+---
+
+[upgrade]: /riak/kv/2.2.0/setup/upgrading/cluster/
+[rolling restarts]: /riak/kv/2.2.0/using/repair-recovery/rolling-restart/
+
+Riak KV functions as a multi-node system, so cluster-level [version upgrades][upgrade] and [restarts][rolling restarts] can be performed on a node-by-node or "rolling" basis.
+
+The following steps should be undertaken on each Riak KV node that you wish to replace:
+
+1\. Create a free node either by:
+
+  a\. Creating an additional node with similar specifications to the other nodes in the cluster.
+
+  b\. Leave a node that is currently in the cluster:
+
+  ```bash
+  riak-admin cluster leave «nodename»
+  ```
+
+  Wait for all transfers to complete:
```

Comment:
So one thing that's hard here is... does "wait..." apply to just leaving or to both?